### PR TITLE
Add initial how-to doc for planning app deployment

### PR DESCRIPTION
**Pull request by chenette**
Signed-off-by: Anne Chenette <chenette@bitwise.io>

### CODE REVIEW

**Comment by davececchi** [comment]
This is a great document.  I'm not sure I see it as a "how-to" - it's more like "Deployment Patterns" in a "Getting Started" section or something.

I could easily see handing this to partners we work with.  Great stuff.

File: docs/howto/planning_splinter_deployment.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+# Network Configuration
+
+The Splinter daemon, splinterd, communicates with other nodes on the network and
+provides a REST API to expose Splinter functions and data to your application.
+The Splinter daemon uses the following ports:
+
+* **8044**: Network endpoint for node-to-node communication
+
+* **8085**: Application communication (encrypted with SSL) via the splinterd
```
Comment (peterschwarz) [inline docs/howto/planning_splinter_deployment.md]: This is not actually true.  The REST API for splinterd is served over standard HTTP.  If the deployment requires HTTPS, the current thinking is that there should be a proxy server for the HTTPS requests.

**Comment by chenette** [comment]
@davececchi - Thanks for the positive feedback!  The "category" for this topic is fluid; it's in `how-to` for now because it's not a concept topic, but can change as more information is developed.

**Comment by chenette** [comment]
FYI, I pushed several fixup commits to add reviewer changes and make images consistent. Key changes:
- Removed "(encrypted with SSL)" from the port 8084 (line 74)
- Added a Summary section to wrap up the document (initial information for now; will expand in later PRs)

File: docs/howto/planning_splinter_deployment.md (hunk 1 above)
Comment (chenette) [reply docs/howto/planning_splinter_deployment.md]: Fixed.

**Pull request merged**